TGM/src/main/java/com/minehut/tgm/modules/kit/KitLoaderModule.java

```diff
@@ -44,7 +45,7 @@ public void load(Match match) {
                 TeamManagerModule teamManagerModule = match.getModule(TeamManagerModule.class);
                 if (kitJson.has("teams")) {
                     for (Object o : kitJson.getAsJsonArray("teams")) {
-                        MatchTeam matchTeam = teamManagerModule.getTeamById((String) o);
+                        MatchTeam matchTeam = teamManagerModule.getTeamById(((JsonPrimitive) o).getAsString());
```

Comment:
the object isn't a string, it's a JsonPrimitive, so i just casted it too primitive then used getAsString()